### DISCUSSION TITLE
Size of axis labels in show_bounds

### DISCUSSION THREAD

**Discussion by ccollins22** [pyvista/pyvista#1944] [Q&A]
Hi, I'm trying to increase the size of the axis labels including the axis numbers. 
I've tried changing font_size in show_bounds but this doesn't seem to change anything:

import pyvista
mesh = pyvista.Sphere()
plotter = pyvista.Plotter()
actor = plotter.add_mesh(mesh)
actor = plotter.show_bounds(grid='front', location='outer', all_edges=True, font_size=30)
plotter.show()

Is this a bug or is there another command controlling this?

**Comment by wayofsamu** [comment]
I had the same problem, this worked for me:
```python
actor.SetUse2DMode(True)
```
It also works if your change the scale to some value bigger than 1:
```python
plotter.set_scale(1.1, 1.1, 1.1)
```
I dont know why, but `SetUse2DMode` is set to `False` if the scale is `[1.0, 1.0, 1.0]`
